data/serializer/src/main/java/tech/pegasys/teku/api/schema/merge/ExecutionPayloadCommon.java

```diff
@@ -39,11 +39,12 @@
   public final Bytes32 parentHash;
 
   @Schema(
+      name = "fee_recipient",
```

Comment:
name never seems to work here. we need to use `@JsonProperty`. this is what got us into trouble renaming things. ideally just call the variable the right name in line with the schema, and lets avoid the mess please.

Reply:
yeah, removed all names and moved to `@JsonProperty`

Reply:
![image](https://user-images.githubusercontent.com/15999009/144840963-f9e422cf-6454-4511-a6e4-5eb2d42880cc.png)

Reply:
@rolfyone I did a cleanup and things looking good to me. By removing names, `Bytes` and `Bytes32` schema references went away as well. I merged it to move things forward. Let me know if you see somethings weird you don't like.

Reply:
that's heaps better. I was thinking I might dig into the other Bytes ones at some point, but now that they're removed it looks like a step in the right direction! thanks.